grails-app/services/au/org/ala/biocollect/merit/WebService.groovy

```diff
@@ -478,11 +486,7 @@ class WebService {
             }
 
             addHubUrlPath(headers)
-            if (useToken) {
-                headers."Authorization" = getAuthHeader()
-            } else {
-                headers."Authorization" = grailsApplication.config.getProperty("api_key")
-            }
+            headers."Authorization" = getAuthHeader()
```

Comment:
can you please add the check to this as well?

Reply:
My bad - I did a replace for `conn.setRequestProperty` and didn't double check! This should be fixed now :)